packages/communication-react/CHANGELOG.beta.md

```diff
@@ -4,31 +4,31 @@ This log was last generated on Tue, 21 May 2024 20:14:27 GMT and should not be m
 
 <!-- Start content -->
 
-## [1.17.0-beta.1](https://github.com/azure/communication-ui-library/tree/@azure/communication-react_v1.17.0-beta.1)
+## [1.17.0-beta.1](https://github.com/azure/communication-ui-library/tree/1.17.0-beta.1)
 
 Tue, 21 May 2024 20:14:27 GMT 
-[Compare changes](https://github.com/azure/communication-ui-library/compare/@azure/communication-react_v1.16.0...@azure/communication-react_v1.17.0-beta.1)
+[Compare changes](https://github.com/azure/communication-ui-library/compare/1.16.0...1.17.0-beta.1)
 
 ### Features
```

Comment:
Looks like this is the first beta release that supports meeting id & passcode (it went straight to stable). Can you add a section for it here. Same as the stable changelog entry

```
### Join Call with Meeting ID/Passcode

We are excited to announce that the Azure Communication Services Web UI Library now supports joining a call with a Meeting Passcode and ID. Currently, Microsoft Teams allows you to require a meeting password for added security and generates a unique meeting ID for each meeting that participants can use to join. This feature now allows users on ACS to join interop Teams calls that have a meeting passcode or an ID generated. This is useful for private calls or events where limited access needs to be granted. Developers can use this functionality today through our composites (e.g CallComposite, CallWithChatComposite) as well as through components.
```